git_machete/tests/functional/test_machete.py

```diff
@@ -672,3 +680,682 @@ def test_help(self) -> None:
+                "current branch 'git machete advance' merges commits from that branch"
+                "and slides out child branches of the downstream branch."
+        )
+        self.assertNotIn(
+            "level-1-branch",
+            self.launch_command("status"),
+            msg="Verify that branch to which advance was performed is removed "
+                "from the git-machete tree and the structure of the git machete "
+                "tree is updated.")
+
+    def test_advance_with_few_possible_downstream_branches_and_yes_option(self) -> None:
+        """Verify behaviour of a 'git machete advance' command.
+
+        Verify that 'git machete advance -y' raises an error when current branch
+        has many synchronized downstream branches and option '-y' is passed.
+
+        """
+        (
+            self.repo_sandbox.new_branch("root")
+            .commit()
+            .new_branch("level-1a-branch")
+            .commit()
+            .check_out("root")
+            .new_branch("level-1b-branch")
+            .commit()
+            .check_out("root")
+        )
+        self.launch_command("discover", "-y")
+
+        # TODO (#120): git-machete exception does sys.exit
+        # with self.assertRaises(
+        #         cmd.MacheteException,
+        #         msg="Verify that 'git machete advance' raises an error when current branch"
+        #             "has many synchronized downstream branches."):
+        #     self.launch_command("advance", '-y')
+
+    def test_update_with_fork_point_not_specified(self) -> None:
+        """Verify behaviour of a 'git machete update --no-interactive rebase' command.
+
+        Verify that 'git machete update --no-interactive rebase' performs
+        'git rebase' to the parent branch of a current branch.
+
+        """
+        (
+            self.repo_sandbox.new_branch("level-0-branch")
+            .commit("Basic commit.")
+            .new_branch("level-1-branch")
+            .commit("Only level-1 commit.")
+            .new_branch("level-2-branch")
+            .commit("Only level-2 commit.")
+            .check_out("level-0-branch")
+            .commit("New commit on level-0-branch")
+        )
+        self.launch_command("discover", "-y")
+
+        parents_new_commit_hash = get_head_commit_hash()
+        self.repo_sandbox.check_out("level-1-branch")
+        self.launch_command("update", "--no-interactive-rebase")
+        new_forkpoint_hash = self.launch_command("fork-point").strip()
+
+        self.assertEqual(
+            parents_new_commit_hash,
+            new_forkpoint_hash,
+            msg="Verify that 'git machete update --no-interactive rebase' perform"
+                "'git rebase' to the parent branch of a current branch."
+        )
+
+    def test_update_with_fork_point_specified(self) -> None:
+        """Verify behaviour of a 'git machete update --no-interactive rebase -f #commit_hash' cmd.
```

Comment:
Nit, the informal convention for placeholders in command line is:
```suggestion
        """Verify behaviour of a 'git machete update --no-interactive rebase -f <commit_hash>' cmd.
```

Reply:
Accepted